docs/crocks/Maybe.md

```diff
@@ -0,0 +1,97 @@
+# Maybe
+
+`Maybe c a`
+
+--
+
+--
+
+```js
+--
+```
+
+`Maybe` exposes the following constructor and instances:
+
+| Constructor | Instance |
+|:---|:---|
+| [`Nothing`](#nothing), [`Just`](#just), [`of`](#of) | [`inspect`](#inspect), [`maybe`](#maybe), [`either`](#either), [`option`](#option), [`type`](#type), [`equals`](#equals), [`coalesce`](#coalesce), [`map`](#map), [`ap`](#ap), [`of`](#of), [`chain`](#chain), [`sequence`](#sequence), [`traverse`](#traverse) |
+
+## Constructors
+
+### Nothing
+
+`Maybe m => c -> m c _`
+
+### Just
+
+`Maybe m => a -> m _ a`
+
+### of
+
+`Maybe m => a -> m _ a`
+
+## Instances
+
+### inspect
+
+`() => String`
+
+### maybe
+
+`Maybe m => m c a ~> a -> m a -> a`
```

Comment:
Well it _was_ a way to extract the value for a [Comonad](https://github.com/fantasyland/fantasy-land#comonad).

But `Maybe` cannot be a `Comonad` because it could be a `Nothing` and it would only be valid for the case:
`Maybe Unit` .

 All other cases would not line up properly, so no `Comonad` for `Maybe`.
(same reason for `Either` and `value` on your last PR)